deepmd/pt/model/model/make_hessian_model.py

```diff
@@ -172,11 +172,10 @@ def _cal_hessian_one_component(
             # fparam: Optional[torch.Tensor] = None,  # nfp
             # aparam: Optional[torch.Tensor] = None,  # (nloc x nap)
             wc = wrapper_class_forward_energy(self, ci, atype, box, fparam, aparam)
-
             hess = torch.autograd.functional.hessian(
                 wc,
                 coord,
-                create_graph=False,
+                create_graph=True,
```

Comment:
Should it be `self.training`?

Reply:
Sure. It has been changed into `create_graph=self.training`.